database/src/seeds/03_basic_project_survey_setup.ts

```diff
@@ -449,6 +453,176 @@ const insertSurveyStakeholderData = (surveyId: number) => `
   ;
 `;
 
+/**
+ * SQL to insert survey sampling site data.
+ *
+ */
+const insertSurveySamplingSiteData = (surveyId: number) =>
+  `INSERT INTO survey_sample_site
+  (
+    survey_id,
+    name,
+    description,
+    geojson,
+    geography
+  ) VALUES (
+    ${surveyId},
+    'Seed Sampling Site',
+    $$${faker.lorem.sentences(2)}$$,
+    '
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Polygon",
+        "coordinates": [
+          [
+            [-121.904297, 50.930738],
+            [-121.904297, 51.971346],
+            [-120.19043, 51.971346],
+            [-120.19043, 50.930738],
+            [-121.904297, 50.930738]
```

Comment:
nit: Can we define a different polygon, just so this one and the survey boundary aren't perfectly overlapping.
Actually, if this could be smaller, and inside the survey boundary, that would be more accurate to real life even.

If you haven't used it before, this site is helpful for plotting/generating geojson: http://geojson.io

Reply:
It would be neat if these values were also generated using faker, but this might be a lot more work than it's worth.

Reply:
I updated the polygon, it's a smaller rectangle within the study area now.